pkg/operator/controller/ingress/deployment.go

```diff
@@ -408,6 +409,27 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 	}
 	env = append(env, corev1.EnvVar{Name: "SSL_MIN_VERSION", Value: minTLSVersion})
 
+	usingIPv4 := false
+	usingIPv6 := false
+	for _, clusterNetworkEntry := range networkConfig.Status.ClusterNetwork {
+		addr, _, err := net.ParseCIDR(clusterNetworkEntry.CIDR)
+		if err != nil {
+			continue
+		}
+		if addr.To4() == nil {
```

Comment:
I think it's a bit easier to understand if the logic is reversed:
```
		if addr.To4() != nil {
			usingIPv4 = true
		} else {
			usingIPv6 = true
		}
```

Reply:
I have reversed it.  Thanks!